io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/JsonInstanceWriter.java

```diff
@@ -31,6 +38,29 @@
  */
 public class JsonInstanceWriter extends AbstractInstanceWriter {
 
+	InstanceToJson instanceToJson;
+	boolean useGeoJsonFeatures = false;
```

Comment:
Make final (and private?) and remove initialization (as it is initialized in the constructors)?

Reply:
Yes as the default constructor will call the parameterised one so no need to initialise it. initialisation removed.